code/src/sixsq/nuvla/server/resources/credential_infrastructure_service_kubernetes.clj

```diff
@@ -0,0 +1,89 @@
+(ns sixsq.nuvla.server.resources.credential-infrastructure-service-kubernetes
```

Comment:
exactly the same as infrastructure-service-swarm. We have to choose if we keep both or rename. We have to  after decision, update existing creds if needed and adapt UI accordingly.

Reply:
Yes, they are the same. I didn't want to address this during the development of the feature. Let's do this as a next step.

Reply:
Created this issue as followup https://github.com/nuvla/api-server/issues/372